src/include/esp-idf/bindings.h

```diff
@@ -61,7 +61,7 @@
 
 #include "driver/adc.h"
 #include "driver/twai.h"
-#ifndef CONFIG_IDF_TARGET_ESP32C3
+#ifndef CONFIG_IDF_TARGET_ESP32C3 || CONFIG_IDF_TARGET_ESP32S3
```

Comment:
Shouldn't this be `if !defined(CONFIG_IDF_TARGET_ESP32C3) && !defined(CONFIG_IDF_TARGET_ESP32S3)`?

Reply:
Oops! Good spot.